package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "private": "false",
   "scripts": {
+    "lint": "eslint .",
     "format": "prettier --write \"./**/*.{js,json,sol,ts}\"",
```

Comment:
Do we still need this?

Reply:
This is a command to run lint over the project, we might add this to a git hook to verify the code. We might also remove this and run ever by `npx eslint .`